app/client/packages/design-system/widgets/src/components/ComboBox/src/ComboBoxTrigger.tsx

```
@@ -0,0 +1,38 @@
+interface ComboBoxTriggerProps {
+  size?: ComboBoxProps["size"];
+  isLoading?: boolean;
+  isInvalid?: boolean;
+  placeholder?: string;
+}
+
+export const ComboBoxTrigger: React.FC<ComboBoxTriggerProps> = (props) => {
+  const { isInvalid, isLoading, placeholder, size } = props;
+
+  const suffix = useMemo(() => {
+    if (Boolean(isLoading)) return <Spinner />;
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)